### PR TITLE
Documentation instances distinction

### PR DESCRIPTION
**Pull request by piomar123**
Making it clear that formatters & console can be accessed only from static module. Relases issue #106

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +207,7 @@ Options
 -------
 
 ``` javascript
-var jsondiffpatch = require('jsondiffpatch').create({
+var jsondiffpatchInst = require('jsondiffpatch').create({
```
Comment (Methuselah96) [inline README.md:210]:
```suggestion
var jsondiffpatchInstance = require('jsondiffpatch').create({
```

**Review by Methuselah96 — CHANGES_REQUESTED** [review]
Thanks for the PR, just a little cleanup to make the naming consistent would be great! Apologies for the delayed response.

**Review by Methuselah96 — APPROVED** [review]

**Comment by Methuselah96** [comment]
Looks like there are merge conflicts, but otherwise good to go!

**Comment by piomar123** [comment]
Resolved conflicts, waiting for CI build

File: README.md [hunk 2]
```diff
@@ -180,8 +180,10 @@ npm install jsondiffpatch
 ```
 
 ```js
-var jsondiffpatch = require('jsondiffpatch').create(options);
+var jsondiffpatch = require('jsondiffpatch');
+var jsondiffpatchInstance = jsondiffpatch.create(options);
 ```
+Some properties are available only from static main module (e.g. formatters, console), so we need to keep the reference to it if we want to use them.
```
Comment (Methuselah96) [inline README.md:186]: The formatter wants a new line here:

```suggestion
var jsondiffpatch = require('jsondiffpatch');
var jsondiffpatchInstance = jsondiffpatch.create(options);
```

Some properties are available only from static main module (e.g. formatters, console), so we need to keep the reference to it if we want to use them.
```

Comment (Methuselah96) [inline README.md:186]: The formatter wants a new line here:

```suggestion

Some properties are available only from static main module (e.g. formatters, console), so we need to keep the reference to it if we want to use them.
```

**Comment by Methuselah96** [comment]
Thanks! Sorry for the back-and-forth for such a simple change. 😄

**Pull request merged**